src/routes/+page.svelte

```diff
@@ -300,7 +300,7 @@
 			<div class="flex items-end justify-end">
 				<Button type="minimal" class="gap-2 text-end text-lg hover-child:translate-x-1">
 					{solutions.slice(-1)[0]?.description ?? ""}
-					<ChevronRight class="h-4 w-4 transition-transform duration-500" />
+					<ChevronRight class="w-10 h-10 transition-transform duration-500" />
```

Comment:
It does affect wider screens :/
<img width="129" alt="CleanShot 2023-08-03 at 21 00 02@2x" src="https://github.com/EmeraldHQ/Website/assets/43064022/9aa5f93e-67f4-4188-8eec-56bda437adfa">

2 options:
- Use Tailwind screen size selectors (`sm:` etc)
- Find out why it gets smaller on smaller screens (better option IMO)

Reply:
The problem is we are using `h-*` so it's used `rem`. It affects all sizes

Reply:
`1rem` is the document font size... it doesn't make sense 🤔

What did you update?